### PR TITLE
Fix: Not starting on new installations

### LINKED ISSUE #256: [Bug] Starts but unable to open a window
**Describe the bug**
Heroic starts, spawns a process but doesnt open a window.

**To Reproduce**
1. Download the deb or appimage from the release section
2. chmod it, (install), run it
3. Nothing except the process spawning happens

**Expected behavior**
The heroic launcher window opens

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 20,04 & Kubuntu 20.04
 - Version 1.5.6 & 1.5.5 (didnt test the others)
-  Game: - 
-  Wine/Proton version: -

**Additional context**
Tried it on an clean install of kubuntu and my main ubuntu. When starting via cli it starts but immediately end itself after.

**Image of the process but no window:**
image

**Image of it starting and ending immediately on my main system**
(ignore the gtk errors, that just my system theme that shouldnt impact it)
image

### PR DESCRIPTION
**Pull request by flavioislima**
Fixed #256 
The bug was introduced with the latest fix I sent related with the custom wine 🤦🏽 . Shit happens.
It happens not only in ubuntu, but all distros since Heroic starts a never-ending loop when looking for custom wine on new installations.
Added a check to skip looking for custom wine/proton when there is no config file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
       "post-checkout": "yarn",
       "post-merge": "yarn",
       "post-rebase": "yarn",
-      "pre-push": "yarn lint && pretty-quick --staged lint-staged",
+      "pre-push": "yarn lint-fix && pretty-quick --staged lint-staged",
```
Comment (adityaruplaha) [inline package.json:135]: This should be reverted as fixes will not be committed and will cause checks to fail. I didn't change this on purpose.

**Review by adityaruplaha — CHANGES_REQUESTED** [review]
One major issue not actually related to this PR, not sure why it was changed.

File: package.json (hunk 1 above)
Comment (flavioislima) [reply package.json:135]: good catch, I didn't want to push this, was just testing 😂

Comment (flavioislima) [reply package.json]: fixed

File: src/components/Settings/index.tsx [hunk 2]
```diff
@@ -1,24 +1,11 @@
-import React, {
-  useEffect,
-  useState
-} from 'react'
+import React, { useEffect, useState } from 'react'
```
Comment (adityaruplaha) [inline src/components/Settings/index.tsx:1]: Yeah we need a standardised styling guideline for these...

File: src/components/Settings/index.tsx [hunk 3]
```diff
@@ -62,7 +49,7 @@ function Settings() {
   const [maxWorkers, setMaxWorkers] = useState(0)
   const [egsPath, setEgsPath] = useState(egsLinkedPath)
   const [language, setLanguage] = useState(
-    () => storage.getItem('language') || ''
+    () => storage.getItem('language') || 'en'
```
Comment (adityaruplaha) [inline src/components/Settings/index.tsx:52]: Good catch, missed this one before.

File: src/components/Settings/index.tsx [hunk 4]
```diff
@@ -72,7 +59,11 @@ function Settings() {
     setOn: setUseGameMode
   } = useToggle(false)
   const { on: showFps, toggle: toggleFps, setOn: setShowFps } = useToggle(false)
-  const { on: offlineMode, toggle: toggleOffline, setOn: setShowOffline } = useToggle(false)
+  const {
```
Comment (adityaruplaha) [inline src/components/Settings/index.tsx:62]: Gonna need standardised style guidelines on this one too....

**Review by adityaruplaha — APPROVED** [review]
PR looks good, left some miscellaneous comments that need to be addressed in upcoming PRs.

File: src/components/Settings/index.tsx (hunk 4 above)
Comment (flavioislima) [reply src/components/Settings/index.tsx:62]: Yeah, the thing is that I'm not sure If the prettier config is working for everyone the same way.
Every time I save a file, something moves. I set up my vscode to format on save, but using the prettier rules we have on the prettierrc file.

**Pull request merged**